py-polars/src/lazyframe/visitor/expr_nodes.rs

```diff
@@ -635,67 +635,86 @@ pub(crate) fn into_py(py: Python<'_>, expr: &AExpr) -> PyResult<PyObject> {
                 propagate_nans,
             } => Agg {
                 name: "min".to_object(py),
-                arguments: input.0,
+                arguments: vec![input.0],
```

Comment:
This can just take `arguments` directly. Not needed to create a new vec.

Reply:
I need to `transmute` I think from `Vec<Node>` to `Vec<usize>` but that should be transparent because the repr is transparent?

Reply:
Oh, nvm. I didn't notice the Node/usize difference. Not worth a transmute this.